src/components/forms/DocumentUploadForm/shared/FileSelected/FileAsImage.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="document-upload-preview">
+    <img
+      :src="imageContent"
+      class="document-upload-preview-img"
+      alt="fileName"
+    >
+    <div
+      class="document-upload-preview-remove"
+      @click.prevent="$emit('file-remove', $event)"
+    >
+      <v-svg-icon
+        name="close"
+        height="10px"
+        width="10px"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import VSvgIcon from '@endpass/ui/kit/VSvgIcon';
+
+export default {
+  name: 'FileAsImage',
+
+  props: {
+    imageContent: {
+      type: String,
+      default: '',
+    },
+
+    fileName: {
+      type: String,
+      default: '',
+    },
+  },
+
+  components: {
+    VSvgIcon,
+  },
+};
+</script>
+
+<style scoped lang="postcss">
+.document-upload-preview {
+  position: relative;
+  max-width: 181px;
+}
+.document-upload-preview-img {
+  max-height: 128px;
```

Comment:
Image can be smaller than `128px`. Add `height: 100%` too.

Reply:
for `<img>` no need to set `height: 100%` it always will be maximize size of element as possible

Reply:
ok